granary/facebook.py

```diff
@@ -579,6 +580,9 @@ def _create(self, obj, preview=None, include_link=False, ignore_formatting=False
           error_plain='No content text found.',
           error_html='No content text found.')
 
+    if obj.get('displayName') and mf2util.is_name_a_title(unicode(obj.get('displayName')), unicode(obj.get('content'))):
+        content = obj.get('displayName') + u"\n\n" + content
```

Comment:
sorry, i was unclear, and also commented on the wrong line. :P should have been the conditional above. `obj.['content']` might be a composite object - examples in `test_microformats.py` - and/or it also might have HTML tags, both of which i think would make `is_name_a_title` return a false negative. `mf2util` does strip punctuation, but not HTML tags. so i think you want something like `microformats2.get_text(obj.get('content'))`. cc @kylewm since he knows mf2util.